Add answer-file generator (#135)
NETWAYS/support-collector: README.md

| redis          | Configuration, logs, packages and service status                                                                                                         |
| webservers     | Includes apache2, httpd and nginx. Collects configuration, logs, packages and service status                                                             |

## Usage

`$ support-collector`

The CLI wizard will guide you through the possible arguments after calling the command.  

> If you prefer to skip the wizard, you can use `--disable-wizard` and use the default control values. A detailed 

Reviewer: A detailed WHAT!

| -f    | --answer-file          | Provide an answer-file to control the collection          |
|       | --disable-wizard       | Disable interactive wizard and use default control values |
|       | --generate-answer-file | Generate an example answer-file with default values       |
| -V    | --verbose              | Enable verbose logging                                    |
| -v    | --version              | Print version and exit                                    |

## Obfuscation

> **WARNING:** Some passwords or secrets are automatically removed, but this no guarantee, so be careful what you share!

With using an answer-file, you are able to add multiple custom obfuscators.  
As these obfuscators are based on regex, you must add a valid regex pattern that meets your requirements.

For example, `Secret:\s*(.*)` will find `Secret: DummyValue` and set it to `Secret: <hidden>`.

In addition, files and folders that follow a specific pattern are not collected. This affects all files that correspond to the following filters:  
`.*`, `*~`, `*.key`, `*.csr`, `*.crt`, and `*.pem`

## Answer File

> With providing an answer-file you can customize the collection by your needs.  

Reviewer: ```suggestion
By providing an answer-file you can customize the data collection.  
```

> In addition to some general control values that customize the collection, further details of modules can also be collected and configured that are not included by default.

Reviewer: ```suggestion
In addition to some general control values that customize the collection, further details for modules - that are not included by default - can be collected and configured.
```

The answer-file has to be in YAML format. 
To generate a default answer-file, you can use `--generate-answer-file`.    

> To provide an answer-file, just use `--answer-file <file>.yml` or `-f <path>.yml`. With using this, the wizard will be skipped.

Reviewer: ```suggestion
To provide an answer-file, just use `--answer-file <file>.yml` or `-f <path>.yml`. Using this, the wizard will be skipped.
```
